config/serverless.yml

```diff
@@ -47,6 +47,11 @@ xpack.cloud_integrations.data_migration.enabled: false
 data.search.sessions.enabled: false
 advanced_settings.enabled: false
 
+# Disable UI of security management plugins
+xpack.security.ui.userManagementEnabled: false
+xpack.security.ui.roleManagementEnabled: false
+xpack.security.ui.roleMappingManagementEnabled: false
```

Comment:
__note:__ fwiw, we're going to drop these flags completely soon and rely on a `buildFlavor` internally instead (https://github.com/elastic/kibana/pull/161930).

Reply:
Okay, thanks for the heads-up!

Reply:
> Would you like me to change this PR so that it uses build flavor instead?

Sorry, missed that part. No worries, we'll make this change in a separate PR (we need to disable both APIs and UIs).